src/extension.ts

```diff
@@ -102,10 +147,11 @@ function newClient(cmd: string, location: string) {
 	}
 	let initializationOptions = {};
 	if (rootModulePaths.length > 0) {
-		initializationOptions = { rootModulePaths };
-	}
-	if (excludeModulePaths.length > 0) {
-		initializationOptions = { excludeModulePaths };
+		initializationOptions = { uuid, rootModulePaths };
```

Comment:
super nitpick, could we have the parameter on the object be just `id`, just trying to avoid enforcing the protocol to have to worry about using a specific uuid library implementation, I'm perfectly happy with `uuid` being used, but would like the variable in `initializationOptions` to imply it just needs to be any unique identifier relative to the client.

Reply:
So not asking the implementation or use of a `uuid` variable name else changes, just at the protocol layer in this if else if else

Reply:
As mentioned in my other review here https://github.com/hashicorp/terraform-ls/pull/279 I would probably focus more on what this is used for than how it is generated, and call this "command prefix"?